### ISSUE TITLE
Finalize: Internet health videos / testing plan

### ISSUE THREAD

**Issue by lovegushwa** [MozillaFoundation/Advocacy#801]
Let's use this ticket to track the final tweaks to the internet health videos. We can loop in Alex and add a revised testing plan tomorrow.

**Comment by hannahkane** [comment]
@lovegushwa - should we put this in the April 14th milestone? Do we need a deadline sooner than that?

**Comment by lovegushwa** [comment]
@hannahkane I'd like to wrap this up tomorrow, let's see where we are EOD, we can move it if needed.

**Comment by lovegushwa** [comment]
@natalieworth @brettgaylor darn nearly final videos are here:
(original with custom subtitles coming tomorrow)

Original (will use facebook auto subtitles)
https://drive.google.com/file/d/0B20-0lpP23c0VjlBc3dEcmNGQ0E/view?usp=sharing

Rapid response version
https://drive.google.com/file/d/0B20-0lpP23c0cDlubUxYcGhMdVE/view?usp=sharing

**Comment by agiedt** [comment]
Comments on the videos and CTAs

I like "Tell Us" as a CTA.

One overall comment is that these videos are long (50 - 60 seconds).  Not too long as in "not engaging", I think they are great, but that is long to get to the CTA.  Is there a way to put the "what do we want the audience to do?" message earlier in the videos.

I'm totally happy to take them and test them to see what kind of engagement we get but I'd suspect that we'll have some issues with getting people to the end and clicking to the madlib.  

So to summarize, I think the CTA is great, but may not get seen.  But, we can test it and see with the understanding that we may need to recut or remix a bit if the clicks to the madlib are lower than desired.

Great job!

**Comment by natalieworth** [comment]
@lovegushwa @hannahkane @brettgaylor @beccaklam Can you all take a look at these final exports before I add in the credits. I want to make sure the timing/ frames are all good before finishing the captions since it's a pain to adjust them after if anything changes. Looking for feedback to make sure sound, spelling, etc. is all good for these. Thanks!
Videos are here:
https://drive.google.com/drive/u/1/folders/0B20-0lpP23c0YmZyZEI0czJCdHM

**Comment by brettgaylor** [comment]
Oops! Sorry I missed this Friday and I'm just on a phone now, so can't look
at finals.

2 things I remember:

The sound mix needs to be adjusted so that when you have quotes/people
talking, the music goes down, and then back up when they are done. Rubber
bands and keyframes are your friend!

I would adjust the quote from Hawking so the text is on screen a little
longer. It's jarring that it leaves right away. I would adjust so just the
"money" part of the guy with his mind exploding is on screen, that way you
could start it later.

Generally I think it's awesome, and we should probs not be fine cutting to
this degree with rapid response vids. But I can give notes forever if you
want!


On Apr 10, 2017 9:18 AM, "Natalie Worth" <notifications@github.com> wrote:

@lovegushwa <https://github.com/lovegushwa> @hannahkane
<https://github.com/hannahkane> @brettgaylor
<https://github.com/brettgaylor> @beccaklam <https://github.com/beccaklam>
Can you all take a look at these final exports before I add in the credits.
I want to make sure the timing/ frames are all good before finishing the
captions since it's a pain to adjust them after if anything changes.
Looking for feedback to make sure sound, spelling, etc. is all good for
these. Thanks!
Videos are here:
https://drive.google.com/drive/u/1/folders/0B20-0lpP23c0YmZyZEI0czJCdHM

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/MozillaFoundation/Advocacy/issues/801#issuecomment-293000392>,
or mute the thread

**Comment by natalieworth** [comment]
@brettgaylor Yes! Thanks! Agree, it's hard to not get into the weeds with the rapid response vids. Should maybe to a mini-debrief or something after? So they don't have to be perfect. Or if there's a way to make them perfect quicker?

**Comment by hannahkane** [comment]
Suuuuuuper tiny thing: at :44 of the long version, the caption of the firefighter image ends with the word "and", and then it's repeated at the beginning of the next frame over the smokestacks.

**Comment by natalieworth** [comment]
@hannahkane thanks! good catch!

**Comment by lovegushwa** [comment]
Additional feedback here: 
https://docs.google.com/document/d/1WGIJj8VyPjgRuXdKKivBR3Of9RjTukKjlZtJrOYxvcA/edit

**Comment by lovegushwa** [comment]
@agiedt here is the revised testing plan:

<img width="1299" alt="screen shot 2017-04-10 at 4 22 55 pm" src="https://cloud.githubusercontent.com/assets/12228435/24886538/1e0454d2-1e0a-11e7-9475-59cca0decc58.png">

**Comment by lovegushwa** [comment]
Final checklist:
- [x] final videos
- [ ] links to multiple versions madlib site for audience testing
- [ ] confirm analytics are all hooked up on mablib site
- [x] transcript for auto subtitles for A. mashup version @natalieworth 

anything missing?

**Comment by agiedt** [comment]
Thanks, I'll be looking at it soon!

A

**Comment by lovegushwa** [comment]
@agiedt no sweat, you can run this test next week. Just making sure you have everything.

**Comment by agiedt** [comment]
I appreciate it.  I'm looking forward to getting into it.  Thanks for
keeping me in the loop.

A

**Comment by natalieworth** [comment]
@lovegushwa @hannahkane @agiedt 
Final videos are up on the google drive here:
https://drive.google.com/drive/u/1/folders/0B20-0lpP23c0SGZtZldDSk9ad1U

1) Mashup (original version): https://drive.google.com/file/d/0B20-0lpP23c0bzZmWmRWWjR5SzA/view
2) Mashup + large captions (rapid response-long version): https://drive.google.com/file/d/0B20-0lpP23c0OU1OTGg3ZXJYaWc/view
3) Text + Stock (rapid response- short version): https://drive.google.com/file/d/0B20-0lpP23c0TWlSNkZ0dTRBLXM/view

**Comment by hannahkane** [comment]
yaaaaaaay!!!! nice work, @natalieworth. :)

**Comment by natalieworth** [comment]
Also for my/ future reference. The working Premiere file is here https://drive.google.com/drive/u/1/folders/0B20-0lpP23c0OGpHalhXTlJZcTA

**Comment by natalieworth** [comment]
@agiedt Are you familiar with uploading guidelines around 'closed caption' files for facebook and twitter? If you are, what kind of file would be best to provide you with for the 1st version of the video (as I exported it without captions inherently in it like the others)? I can put the text into a google document, but just wanted to check if you need anything on your end for the closed caption files?

**Comment by natalieworth** [comment]
Hey @agiedt , I've exported some poster images for each of the different videos. Let me know if you think these would be good for social? I liked the ominous look of the computer screen, but the 3rd video I included some different options. Thanks :)
Video 1:
![01-internet-health-long-version-no-cc](https://cloud.githubusercontent.com/assets/14876567/25099862/d26edfde-2362-11e7-8d67-5233d4fb974b.jpg)

Video 2:
![02-rapid-response-long-version-with-cc](https://cloud.githubusercontent.com/assets/14876567/25099863/d80033ee-2362-11e7-999c-a0daddc7683b.jpg)

Video 3 (I exported a couple options for this, I felt like it may be interesting to test a different image compared to the computer one above?):
a)
![03-rapid-response-short-version-with-cc-v1](https://cloud.githubusercontent.com/assets/14876567/25099867/defe33ee-2362-11e7-9440-73d716a172fc.jpg)
b)
![03-rapid-response-short-version-with-cc-v2](https://cloud.githubusercontent.com/assets/14876567/25099868/df002eb0-2362-11e7-961c-b80dccfdee61.jpg)
c)
![03-rapid-response-short-version-with-cc-v3](https://cloud.githubusercontent.com/assets/14876567/25099869/df00fdcc-2362-11e7-89f0-c5349eec596d.jpg)

cc @lovegushwa @caltrider @beccaklam @hannahkane

**Comment by lovegushwa** [comment]
It seems like it would be better if we tested thevideos with the same poster so it's more of a test of the format and less about the poster ... @agiedt what do you think? 

CC @natalieworth

**Comment by agiedt** [comment]
If we make st desire a read on the videos then Yes, one poster/image and
copy, but if the poster and copy don't generate sufficient response we
should be ready to traffic a second image and copy.

**Comment by natalieworth** [comment]
@lovegushwa @agiedt @beccaklam @hannahkane Uploaded 2 poster options here for the video (kept them the same for testing sake):
![01-internet-health-long-version-no-cc](https://cloud.githubusercontent.com/assets/14876567/25297876/805a13a8-26a5-11e7-906f-f2d6ff72c829.jpg)
![01-internet-health-long-version-no-cc](https://cloud.githubusercontent.com/assets/14876567/25297881/837e5a80-26a5-11e7-8374-3f8e008452d5.jpg)

@agiedt I also added the Closed Caption text document that will be needed for uploading to facebook:
https://docs.google.com/document/d/17Ru_fbNQl6Wg3Nxn6YvrGfJtZhni47h8i78EA4HqN7o/edit

**Comment by agiedt** [comment]
Thanks for the CC stuff

**Comment by lovegushwa** [comment]
@agiedt let's start with "is the internet sick?" as the poster. 
https://drive.google.com/file/d/0B20-0lpP23c0LWpSNzhBMTZFNE0/view?usp=sharing

If its cropped weird for [video 2](https://drive.google.com/file/d/0B20-0lpP23c0OU1OTGg3ZXJYaWc/view) then you can use this alternate version: https://drive.google.com/file/d/0B2qu3ExKWIlvUXpLVWtIamdvZkU/view?usp=sharing

**Comment by agiedt** [comment]
Thanks!

**Comment by HPaulJohnson** [comment]
To help keep track of all the different components of this test, a matrix to rule them all: 

https://docs.google.com/spreadsheets/d/1X0YjIOSrziJXJa5beBCrsWeFm9clUc0JflJ12YrVu3A/edit?ts=58f93f33#gid=233054778

**Comment by hannahkane** [comment]
Can we close this ticket?

**Comment by HPaulJohnson** [comment]
Yes, this is live.
